webapp/params.go

```diff
@@ -61,23 +61,15 @@ func ParseBrowserParam(r *http.Request) (browser string, err error) {
 	return "", fmt.Errorf("Invalid browser param value: %s", browser)
 }
 
-// ParseBrowsersParam returns a sorted list of browsers to include.
-// It parses the 'browsers' parameter, split on commas, and also checks for the (repeatable) 'browser' params,
-// before falling back to the default set of browsers.
+// ParseBrowsersParam returns a sorted list of browser params for the request.
+// It parses the 'browsers' parameter, split on commas, and also checks for the (repeatable)
+// 'browser' params.
 func ParseBrowsersParam(r *http.Request) (browsers []string, err error) {
 	browsers = r.URL.Query()["browser"]
 	if browsersParam := r.URL.Query().Get("browsers"); browsersParam != "" {
 		browsers = append(browsers, strings.Split(browsersParam, ",")...)
 	}
-	// If no params found, return the default.
-	var browserNames []string
-	if browserNames, err = GetBrowserNames(); err != nil {
-		return nil, err
-	}
-	if len(browsers) == 0 {
-		return browserNames, nil
-	}
-	// Otherwise filter to valid browser names.
+	// Otherwise validate browser names.
```

Comment:
This comment doesn't make sense now that there's no return in the preceding if clause.

Reply:
Done.